README.md

```diff
@@ -473,6 +475,7 @@ Notable projects using uhubctl
 | [Do it yourself PPPS](https://git.io/J3lHs)              | Solder wires in your USB hub to support uhubctl         |
 | [Open source PPPS hub](https://tinyurl.com/yckhystt)     | Open source hardware project for uhubctl compatible hub |
 | [Python Wrapper for uhubctl](https://github.com/nbuchwitz/python3-uhubctl) | Module to use uhubctl with Python     |
+| [Wrapper for uhubctl](https://tinyurl.com/2s352y4n)      | Specify topology of USB hubs to allow complex setups    |
```

Comment:
Please split this to different PR - adding to list of devices is not the same as adding to list of notable projects.
Also, when you create separate PR, please call it Python Wrapper, not just wrapper. Since there are already two other python wrappers (including different one with the same name as yours: [uhubctl](https://github.com/DuraTech-Industries/pyhubctl)), it is good idea to mention that too.

Reply:
Thanks a lot for your feedback! I update this PR.